source/config.js

```diff
@@ -4,11 +4,7 @@
  * Configuration
  */
 
-var secure = window.location.protocol === 'https:';
-
 module.exports = {
     name: 'likely',
-    prefix: 'likely__',
-    secure: secure,
-    protocol: secure ? 'https:' : 'http:',
+    prefix: 'likely__'
```

Comment:
Weird. We require a dangling comma here, and it looks like Travis doesn’t catch this. Most likely this is happening because we run ESLint with `--fix` option, so it fixes the code on Travis and exits with `0`. I’ll update the scripts today, but until they’re fixed, could you please run `npm run codestyle` on your own?